app/components/live-participants.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+import { ROLES } from '../constants/live';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+
+export default class LiveParticipantsComponent extends Component {
+  @service live;
```

Comment:
remove this @service as we're using getter for getting service below

Reply:
fixed